docs/design/3_1_6_base_station.tex

```diff
@@ -13,10 +13,19 @@ \subsubsection{Device Alternatives}
 
 One functionality that mobile devices lack is the support for common machine learning platforms (ex. TensorFlow). As the client may want to perform additional ML processing on the base station for further analysis in the future, laptops were ultimately selected as the base station's platform. Support for smart devices (to facilitate multiple viewers) may be added if time permits.
 
-\subsubsection{Display}
+\subsubsection {Graphical User Interface}
+A web-application is used as the GUI for the base station. This allows for universal compatibility with laptops and smart mobile devices, regardless of their operating system.
```

Comment:
You can probably just roll this into a the next section, this section is too short imo.